tests/unit/test_events.py

```diff
@@ -174,6 +188,65 @@ def test_parse_release(self, github_release_webhook):
             event_object.project_url == "https://github.com/packit-service/hello-world"
         )
 
+    def test_parse_mr(self, merge_request):
+        event_object = Parser.parse_event(merge_request)
+
+        assert isinstance(event_object, MergeRequestGitlabEvent)
+        assert event_object.trigger == TheJobTriggerType.pull_request
+        assert event_object.action == GitlabEventAction.opened
+        assert event_object.object_id == 58759529
+        assert event_object.object_iid == 1
+        assert event_object.source_repo_namespace == "testing-packit"
+        assert event_object.source_repo_name == "hello-there"
+        assert event_object.commit_sha == "1f6a716aa7a618a9ffe56970d77177d99d100022"
+        assert event_object.target_repo_namespace == "testing-packit"
+        assert event_object.target_repo_name == "hello-there"
+        assert event_object.https_url == "https://gitlab.com/testing-packit/hello-there"
+
+        assert isinstance(event_object.project, GitlabProject)
+        assert event_object.project.full_repo_name == "testing-packit/hello-there"
+
+        flexmock(PackageConfigGetter).should_receive(
+            "get_package_config_from_repo"
+        ).with_args(
+            base_project=None,
+            project=event_object.project,
+            pr_id=1,
+            reference="1f6a716aa7a618a9ffe56970d77177d99d100022",
+            fail_when_missing=False,
+            spec_file_path=None,
+        ).and_return(
+            flexmock()
+        ).once()
+        assert event_object.package_config
+
+    def test_parse_mr_action(self, merge_request_update):
+        event_object = Parser.parse_event(merge_request_update)
+        assert isinstance(event_object, MergeRequestGitlabEvent)
+        assert event_object.action == GitlabEventAction.update
+        assert event_object.commit_sha == "45e272a57335e4e308f3176df6e9226a9e7805a9"
+        assert event_object.object_iid == 2
+
+        assert isinstance(event_object.project, GitlabProject)
+        assert event_object.project.full_repo_name == "testing-packit/hello-there"
+        assert (
+            not event_object.base_project  # With Github app, we cannot work with fork repo
```

Comment:
Another copy-paste leftover from GitHub..;)

Reply:
Made the changes.